engineapi/engineapi/models.py

```diff
@@ -155,14 +157,87 @@ class DropperClaimant(Base):  # type: ignore
     )
 
 
+class CallRequestType(Base):  # type: ignore
+    """
+    CallRequestType contains versions of call requests like:
+    raw or dropper-v0.2.0.
+    """
+
+    __tablename__ = "call_request_types"
+
+    id = Column(
+        UUID(as_uuid=True),
+        primary_key=True,
+        default=uuid.uuid4,
+        unique=True,
+    )
+    request_type = Column(VARCHAR(128), nullable=False, unique=True, index=True)
+    description = Column(String, nullable=True)
+
+    call_requests = relationship(
+        "CallRequest",
+        back_populates="call_request_type",
+        cascade="all, delete, delete-orphan",
+    )
+
+
+class MetatxHolder(Base):  # type: ignore
```

Comment:
I think it's better to call this `MetatxRequester` not `MetatxHolder` - they are not holding anything, they are making a request to someone else to execute a function on a smart contract.

Reply:
Done